### PR TITLE
Development: First project complete

### PR DESCRIPTION
**Pull request by luccamicale**
Mi First project

.I added specific models following the instructions
.It is necessary for the future to add more styles.
.Improve this codding in the next steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,28 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
+# Hi Microverse!!
 
 > Description the project.
```
Comment (Nemwel-Boniface) [inline README.md]: As part of documenting your project professionally, the README.md file is the key to this. The following areas seem to not have been addressed as you have deleted them from the file.
- [ ] The top Microverse tag that is required in all Microverse projects is to be present. Kindly re-add this at the top  ``````
- [ ] The "Built with" section is empty. Kindly list what you used to build this project. eg HTML, CSSS, linters etc
- [ ] Kindly customize your MIT.md file to include your details that are relevant for this specific project

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Nemwel-Boniface) [inline index.html:7]: - [ ] Kindly give your project an appropriate title. Giving it just "title" does not help your viewers know what this website is about.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @luccamicale            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Tou have made a great effort in setting up your project so far :clap: 
Welcome to Microverse! :partying_face: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [x] According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) and from [documenting your project professionally](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) texts, you are required to document your project professionally. The following areas still have not met the requirements:

1. Your pull request title is not descriptive of what this pull request is for. "Development is rather very vague"
2. You do not have a pull request description. Assume you were the reviewer, how would you know what changes this person made and wants you to review?

- [x] I have also realized that you have not set up the project linters as per the project requirements. Visit [this link](https://github.com/microverseinc/linters-config/tree/master/html-css) and follow the steps that are provided there to set up the correct linters for this project.
- [x] I am also not able to see the .gitignore in your files list, this is where you are required to ignore both the test.md file and the node_modules (packages and dependencies of your project). Kindly add them.

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luccamicale** [comment]
was able to include the linters in my project, improve my titles and presentation and add a .gitignore folder.

**Review by frank1738 — APPROVED** [review]
Hi  @luccamicale , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

5SW



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**